test/helpers/Config.ts

```diff
@@ -15,14 +16,17 @@ export const metadataConfig = {
 export const execOpts = {
   env: {
     ...process.env,
-    NODE_URL: 'http://localhost:8545',
+    NODE_URL: `${process.env.NODE_URL}` || 'http://localhost:8545',
     TOKEN_ADDRESS: '0x0000000000000000000000000000000000000000',
```

Comment:
This too should be passed through an env for setting `cUSD` for instance

Reply:
good point, fixed